_examples/in-cluster/main.tf

```diff
@@ -0,0 +1,85 @@
+variable "minikube_host_ip" {}
+variable "minikube_target_ip" {}
+
+provider "kubernetes" {
+  version = "1.10.0"
+
+  host                   = "https://${var.minikube_host_ip}:8443"
+  client_certificate     = file("~/.minikube/client.crt")
+  client_key             = file("~/.minikube/client.key")
+  cluster_ca_certificate = file("~/.minikube/ca.crt")
+}
+
+resource "kubernetes_config_map" "terraform" {
+  metadata {
+    name = "terraform"
+  }
+
+  data = {
+    "main.tf" = <<-EOT
+      provider "kubernetes" {
+        version = "1.10.1-dev"
+      }
+
+      resource "kubernetes_namespace" "test" {
+        metadata {
+          name = "test"
+        }
+      }
+    EOT
+  }
+}
+
+resource "kubernetes_job" "terraform" {
+  metadata {
+    name = "terraform"
+  }
+  spec {
+    backoff_limit = 1
+    template {
+      metadata {}
+      spec {
+        container {
+          name  = "terraform"
+          image = "hashicorp/terraform:0.12.13"
+          command = [
+            "sh",
+            "-c",
+            "set -x && apk --no-cache add curl && mkdir /tf && cd /tf && cp /configuration/main.tf . && mkdir -p ~/.terraform.d/plugins && curl https://storage.googleapis.com/pdecat-builds/terraform-provider-kubernetes_v1.10.1-dev > ~/.terraform.d/plugins/terraform-provider-kubernetes_v1.10.1-dev && chmod +x ~/.terraform.d/plugins/* && terraform init && terraform plan && terraform apply -auto-approve && sleep 10 && terraform destroy -auto-approve"
```

Comment:
This may be reverted to https://github.com/terraform-providers/terraform-provider-kubernetes/pull/686/commits/726e4a038cdc7f4938f80d93139eeef46fe1e5f7#diff-52766c12b9ab0b44de46fcc3af48a543R45 before merging.